BeardedSpice/BSTrack.m

```diff
@@ -18,19 +18,21 @@
 NSString *const kBSTrackNameFavorited = @"favorited";
 NSString *const kBSTrackNameIdentifier = @"BSTrack Notification";
 
+const NSString *kImageLock = @"kImageLock";
```

Comment:
This is an NSString const, which as a class doesn't make sense. We want the pointer to be const and unchangeable. https://stackoverflow.com/a/15157226